### DISCUSSION TITLE
Option to only import newly downloaded things.

### DISCUSSION THREAD

**Discussion by inzork** [ppy/osu#28352] [Ideas] [answered]
I download every ranked map in stable and I only play lazer every couple of weeks. 
It would be nice if we had an extra option that only looked at stuff that you downloaded in stable after the last import date, so I wouldn't have to wait for all 32,000 beatmaps to get added again.

Attached scuffed "concept art" for reference.
![a](https://github.com/ppy/osu/assets/103394454/c5f3e19d-bdc1-4c87-a00a-73e9303a8e17)

**Comment by SupDos** [comment]
I believe it does skip anything that was previously imported already (as per https://github.com/ppy/osu/discussions/19024#discussioncomment-3098301) meaning anything that is already in lazer isn't imported again

**Reply by inzork** [reply]
I mean yeah but the real problem is, I still have to wait for every map to get checked if it was imported before. I also noticed they fixed the double import thing with skins cuz if you imported them 2x every skin was listed 2x.

**Comment by peppy** [comment, marked as answer]
This shouldn't be something you do often. But I do agree that we could likely improve performance with some local optimisations (skipping existing beatmaps based on modified date or something similar, rather than full hash / online checks).

**Reply by inzork** [reply]
To be honest, I do it this way because I'm lazy. Currently processing this many maps takes like 1 hour+ but it's still more convenient than going into direct and manually clicking download on 100+ maps, I just do something else in the meantime.
